module.json

```diff
@@ -119,24 +132,25 @@
         "readOnly": true
       }
     },
-    "Next Event": {
+    "messageTimer": {
```

Comment:
Something is going wrong in your merging
make sure to that the main branch is up to date and then create a branch from that to work on